Testscripts/Windows/PERF-XDP-TCP-LATENCY.ps1

```diff
@@ -0,0 +1,205 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+
+<#
+.Description
+    This script deploys the VM and verifies there are no regression in Network Latency caused
+    by XDP. We achieve this by comparing lagscope results.
+#>
+
+param([object] $AllVmData,
+    [object] $CurrentTestData)
+
+$iFaceName = "eth1"
+# Threshold value (40%) is calculated by analyzing 10 samples of latency values with and w/o XDP
+$thresholdValue = 1.4
+
+function Run_Lagscope_PERF {
+    $ResultDir = $args[0]
+    Write-LogInfo "Starting lagscope perf test"
+    $testJob = Run-LinuxCmd -ip $receiverVMData.PublicIP -port $receiverVMData.SSHPort `
+        -username $user -password $password -command "./perf_lagscope.sh &> ~/lagscopeConsoleLogs.txt" `
+        -RunInBackground -runAsSudo
+    while ((Get-Job -Id $testJob).State -eq "Running") {
+        $currentStatus = Run-LinuxCmd -ip $receiverVMData.PublicIP -port $receiverVMData.SSHPort `
+            -username $user -password $password -command "tail -2 ~/lagscopeConsoleLogs.txt | head -1" -runAsSudo
+        Write-LogInfo "Current Test Status: $currentStatus"
+        Wait-Time -seconds 20
+    }
+    # Copy result
+    Copy-RemoteFiles -downloadFrom $receiverVMData.PublicIP -port $receiverVMData.SSHPort `
+        -username "root" -password $password -download `
+        -downloadTo $ResultDir -files "/root/lagscope-n*.txt"

```

Comment:
don't rely on that you (the test account) have permission to access /root.

Reply:
fixed it.